src/api.tsx

```diff
@@ -30,10 +30,12 @@ export const scrollIntoView = async (
     align,
   );
 
-  scrollView
-    .getScrollResponder()
-    // @ts-ignore
-    .scrollResponderScrollTo({ x: 0, y: newScrollY, animated });
+  const scrollResponder = scrollView.getScrollResponder();
+  if (scrollResponder.scrollResponderScrollTo != null) {
+    scrollResponder.scrollResponderScrollTo({ x: 0, y: newScrollY, animated });
+  } else {
+    scrollView.scrollTo({ x: 0, y: newScrollY, animated });
```

Comment:
are you sure this works?

Because another PR with much more attention and testing from the community is calling `scrollTo` on the responder, not on the scrollView.

Maybe both work, but can you double-check please?

Reply:
https://github.com/APSL/react-native-keyboard-aware-scroll-view/pull/510/files

Reply:
going to move on and merge this, at worst it would only not work on recent RN versions so not worst than current state